lib/Zonemaster/Backend/DB/PostgreSQL.pm

```diff
@@ -100,7 +100,7 @@ sub create_db {
     # TEST RESULTS
     ####################################################################
     $dbh->do(
-        'CREATE TABLE test_results (
+        'CREATE TABLE IF NOT EXISTS test_results (
```

Comment:
Couldn't it be enough to check if the table exists, and if not create all tables, triggers and indexes?